Automattic/wp-calypso
# Sites Page: Prevent duplicate 'Business' from appearing when expired

Bug introduced in https://github.com/Automattic/wp-calypso/pull/74646

## Proposed Changes

Prevents duplicate 'Business' from appearing when plan is expired:

<img width="473" alt="image" src="https://user-images.githubusercontent.com/36432/231304480-c8268006-e92b-4144-8694-ce82c9004ffa.png">

These two conditionals aren't equivalent:
* https://github.com/Automattic/wp-calypso/pull/74646/files#diff-be436b2097d20ffc385e8ef2a2b5cdc160b528ece9475d22f9ca291ecde650f8R44-R53
* https://github.com/Automattic/wp-calypso/pull/74646/files#diff-0ab55e20b31ca28d1f43bea293ab55c2e8b414ec7be2334cc4ff22461bd1cb8eL153-L163

## Testing Instructions

1. View `/sites` with a variety of sites and verify the plans appear as expected.
